themes/digital.gov/src/scss/new/guides/_guide-landing.scss

```diff
@@ -0,0 +1,60 @@
+@use "uswds-core" as *;
+
+.dg-guides {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  padding-top: units(2);
+
+  @include at-media("tablet") {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @include at-media("desktop") {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+.dg-guides-card {
+  border: 2px color("gray-cool-10") solid;
```

Comment:
Is there a theme color we could use instead of `gray-cool-10`?

Reply:
Yes, replaced it with `base-lightest` which is one step lighter.